packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -94,11 +95,28 @@ abstract class HttpRequestData extends NetworkRequest {
     if (mime.startsWith('[')) {
       mime = mime.substring(1);
     }
+    if (mime.endsWith(']')) {
+      mime = mime.substring(0, mime.length - 1);
+    }
     // TODO(kenz): consider special casing some extensions. For example,
```

Comment:
TODO I'm referring to is right here :)

Reply:
I'm blind... :-) Added handling for `text/plain` while I'm here.